cdm/src/main/java/thredds/client/catalog/Catalog.java

```diff
@@ -136,6 +142,18 @@ private Service findService(List<Service> services, String want)  {
     return null;
   }
 
+  private Service findService(List<Service> services, ServiceType type)  {
```

Comment:
hmmm, dont remember those...

On Wed, Mar 2, 2016 at 2:28 PM, Ryan May <notifications@github.com> wrote:

> In cdm/src/main/java/thredds/client/catalog/Catalog.java
> <https://github.com/Unidata/thredds/pull/461#discussion_r54794514>:
>
> > @@ -136,6 +142,18 @@ private Service findService(List<Service> services, String want)  {
> >      return null;
> >    }
> >
> > +  private Service findService(List<Service> services, ServiceType type)  {
>
> I only ask because I know we had quite a few Coverity issues like this. I
> don't have a stake in it, other than to prevent the Coverity # going up.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Unidata/thredds/pull/461/files#r54794514>.
>